ScatterGather/ScatterGather/ViewController.swift

```diff
@@ -0,0 +1,178 @@
+                    }
+                }
+                UIView.addKeyframe(withRelativeStartTime: 0.8, relativeDuration: 0.2) {
+                    self.currentRotationValues = []
+                    for label in self.letterLabels {
+                        let newRotationAngle = (CGFloat(Int.random(in: 1...20)) / 10) * .pi
+                        self.currentRotationValues.append(newRotationAngle)
+                        label.transform = CGAffineTransform(rotationAngle: newRotationAngle)
+                        label.center.x = CGFloat(Int.random(in: 0...Int(self.view.bounds.size.width)))
+                        label.center.y = CGFloat(Int.random(in: Int(self.view.bounds.size.height * 0.8)...Int(self.view.bounds.size.height)))
+                    }
+                }
+                
+            }
+            
+            UIView.animateKeyframes(withDuration: 4, delay: 0, options: [], animations: animBlock, completion: nil)
+            
+            isScattered = true
+            
+        case true:
+            UIView.animate(withDuration: 2) {
+                self.logoImageView.alpha = 1
+                for label in self.letterLabels {
+                    label.layer.backgroundColor = UIColor(hue: 349/359, saturation: 0.90, brightness: 0.73, alpha: 1).cgColor
+                    label.textColor = UIColor(hue: 180/359, saturation: 0, brightness: 1, alpha: 1)
+                    label.shadowColor = UIColor(hue: 349/359, saturation: 1, brightness: 0.5, alpha: 1)
+                }
+            }
+            self.takeYourPositions()
+            isScattered = false
+        }
+    }
+    
+    func setUpView() {
+        let letters = ["L", "A", "M", "B", "D", "A"]
+
+        for i in 0...5 {
+            let label = UILabel(frame: CGRect(x: self.view.center.x, y: 0, width: 55, height: 55))
+            label.layer.masksToBounds = true
+            label.layer.cornerRadius = 12
+            label.text = letters[i]
+            label.textAlignment = .center
+            label.font = UIFont(name: "Noteworthy-Bold", size: 40)
+            label.layer.backgroundColor = UIColor(hue: 349/359, saturation: 0.90, brightness: 0.73, alpha: 1).cgColor
+            label.textColor = UIColor(hue: 180/359, saturation: 0, brightness: 1, alpha: 1)
+            label.shadowColor = UIColor(hue: 349/359, saturation: 1, brightness: 0.5, alpha: 1)
+            label.shadowOffset = CGSize(width: 3, height: 3)
+            view.addSubview(label)
+            label.widthAnchor.constraint(equalTo: label.heightAnchor).isActive = true
+            letterLabels.append(label)
+        }
+        
+        self.logoImageView.alpha = 0
+        UIView.animate(withDuration: 2) {
+            self.logoImageView.alpha = 1
+        }
+    }
+    
+    func takeYourPositions() {
+        let xModifier: [CGFloat] = [-150, -90, -30, 30, 90, 150]
```

Comment:
Just showing you another way to make an array of numbers that step by a certain amount:

``` swift
let xModifier: [CGFloat] = Array(stride(from: -150, to: 151, by: 60))
```

Note the end is non-inclusive, and in this case it could be 200 and it wouldn't make a difference as we are striding by 60, so another number would not be produced.